sdks/typescript/client/package.json

```diff
@@ -19,11 +19,14 @@
     "dist"
   ],
   "dependencies": {
-    "@modelcontextprotocol/sdk": "^1.22.0",
+    "@modelcontextprotocol/ext-apps": "^0.2.0",
+    "@modelcontextprotocol/sdk": "^1.24.0",
+    "@mcp-ui/shared": "workspace:*",
```

Comment:
@ochafik was this change needed? it is causing a resolution issue when trying to install locally `➤ YN0001: │ Error: @mcp-ui/shared@workspace:*: Workspace not found (@mcp-ui/shared@workspace:*)` removing this line fixes the error